framework/Cargo.toml

```diff
@@ -52,11 +52,10 @@ protobuf = { version = "2", features = ["with-bytes"] }
 clap = { version = "4.0.32", features = ["derive"] }
 semver = "1.0"
 cw-semver = { version = "1.0" }
-# cw-orch = { version = "0.16.1" }
-cw-orch = { version = "0.16.3" }
+cw-orch = { version = "0.18.1" }
 tokio = { version = "1.4", features = ["full"] }
-polytone = { git = "https://github.com/AbstractSDK/polytone", tag = "v1.0.0" }
-polytone-note = { git = "https://github.com/AbstractSDK/polytone", tag = "v1.0.0" }
+polytone = { git = "https://github.com/AbstractSDK/polytone", rev = "2849001" }
```

Comment:
Can you create a tag on the fork instead of using a commit tag?

Reply:
Done